onnx2pytorch/operations/slice.py

```diff
@@ -2,6 +2,26 @@
 from torch import nn
 
 
+def _to_positive_step(orig_slice, N):
+    # Get rid of backward slices
```

Comment:
Can you please add a short docstring what this function does and in what case it is used?

Reply:
Added a docstring -- Pytorch still doesn't support slicing tensors with a negative step (gives "step must be greater than zero").